src/AsyncCreatable.js

```diff
@@ -7,19 +7,42 @@ const AsyncCreatable = React.createClass({
 	render () {
 		return (
 			<Select.Async {...this.props}>
-				{(asyncProps) => (
-					<Select.Creatable {...this.props}>
-						{(creatableProps) => (
-							<Select
-								{...asyncProps}
-								{...creatableProps}
-							/>
-						)}
+				{({ref, ...asyncProps}) => {
+					const asyncRef = ref
+					return <Select.Creatable {...asyncProps} >
+						{({ref, ...creatableProps}) => {
+							const creatableRef = ref
+							return this.props.children({
+								...creatableProps,
+								ref: (select) => {
+									creatableRef(select)
+									asyncRef(select)
+									this.select = select
+								}
+							})
+						}}
```

Comment:
Cool, think I'm following now. 
Somewhat unrelated do we want to allow consumer to be able to assign its own child, shouldn't the last child always be a <Select /> component?